README.md

```diff
@@ -1,140 +1,111 @@
+[![Build Status](https://github.com/postmodern/command_kit.rb/workflows/CI/badge.svg?branch=main)](https://github.com/postmodern/command_kit.rb/actions)
+
 # command_kit
 
+A Ruby toolkit for building clean, correct, and robust CLI commands as
+plain-old Ruby classes.
+
 * [Homepage](https://github.com/postmodern/command_kit.rb#readme)
-* [Forum](https://github.com/postmodern/command_kit.rb/discussions)
-* [Issues](https://github.com/postmodern/command_kit.rb/issues)
+* [Forum](https://github.com/postmodern/command_kit.rb/discussions) / [Issues](https://github.com/postmodern/command_kit.rb/issues)
 * [Documentation](http://rubydoc.info/gems/command_kit/frames)
 
-## Description
+## Install
```

Comment:
Eh, I tend to try to sell the reader on the project first then show how to install it. (Description -> Features -> Example Code -> Install). Maybe Examples could go below Install.